airbyte-lib/pyproject.toml

```diff
@@ -50,6 +50,7 @@ types-requests = "2.31.0.4"
 freezegun = "^1.4.0"
 airbyte-source-faker = "^6.0.0"
 viztracer = "^0.16.2"
+tomli = "^2.0"
```

Comment:
Is `tomli` used? I couldn't see references to it, so I am guessing perhaps it was used in a previous version? If not needed, we can delete.